src/main/java/uk/gov/ons/ctp/response/sample/service/SampleService.java

```diff
@@ -79,18 +79,19 @@
    */
   PartyDTO sendToPartyService(PartyCreationRequestDTO party) throws Exception;
 
-  Optional<SampleSummary> failSampleSummary(SampleSummary sampleSummary, String message);
-
-  Optional<SampleSummary> failSampleSummary(SampleSummary sampleSummary, Exception exception);
-
   /**
-   * Ingest Survey Sample
-   *
+   * Ingest survey sample
+   * @param sampleSummary a newly created samplesummary
    * @param file Multipart File of SurveySample to be used
    * @param type Type of Survey to be used
-   * @throws Exception exception thrown
+   * @return an updated samplesummary
+   * @throws Exception
```

Comment:
Might be worth added the scenario it raises an exception in the docs?

Reply:
Done